examples/gallery/lines/line_custom_cpt.py

```diff
@@ -21,8 +21,12 @@
 fig = pygmt.Figure()
 fig.basemap(frame=["WSne", "af"], region=[20, 30, -10, 10])
 
-# Create a custom CPT with the batlow CPT and 10 discrete z-values (colors)
-pygmt.makecpt(cmap="batlow", series=[0, 10, 1])
+# Create a custom CPT with the batlow CPT and 10 discrete z-values (colors),
+# use color_model="+c" + ','.join(map(str, range(1,11))) to write the color
+# palette in categorical format and add labels (1) to (10) for the colorbar legend
+pygmt.makecpt(
+    cmap="batlow", series=[0, 9, 1], color_model="+c" + ",".join(map(str, range(1, 11)))
```

Comment:
Two comments about the `color_model` parameter:

1. the `zvalue` passed to `Figure.plot()` starts from 0, so the colorbar annotations should also start from 0
2. If I understand it correctly, `+c0-9` works the same way to `+c0,1,2,3,4,5,6,7,8,9`. The former is much simpler.

```suggestion
    cmap="batlow", series=[0, 9, 1], color_model="+c0-9")
```